### PR TITLE
Change learning rate decay to exponential, update network config/hyperparam

### PR DESCRIPTION
**Pull request by yanii**
Using exponential learning rate decay makes it easier to compare different runs, avoid having to manually figure out where best steps are, etc. 

Also switch hyperparameters/network config to best found so far - get val RMS of 8.616

image

### CODE REVIEW

**Review by tom-roddick — APPROVED** [review]
Oo nice work, well done on getting the error down! Approving!
